flow/project.py

```diff
@@ -1079,11 +1014,59 @@ def print_status(self, jobs=None, overview=True, overview_max_lines=None,
         :param param_max_width:
             Limit the number of characters of parameter columns,
             see also: :py:meth:`~.update_aliases`.
+        :type param_max_width:
+            int
+        :param expand:
+            Present labels and operations in two separate tables.
+        :type expand:
+            bool
+        :param all_ops:
+            Include operations that are not eligible to run.
+        :type all_ops:
+            bool
+        :param only_incomplete:
+            Remove all jobs that have no eligible operations.
```

Comment:
"Only show jobs that have eligible operations."

The "remove jobs" is misleading.